src/main/java/seedu/address/model/student/Year.java

```diff
@@ -38,7 +39,6 @@ public static boolean isValidYear(String test) {
         return test.matches(VALIDATION_REGEX);
     }
 
-
     @Override
     public String toString() {
         return String.valueOf(year);
```

Comment:
Since `year` is just a String now, `toString()` could just return `year` itself. But this is also really minor.

Reply:
Agreed. I have made the changes